examples/streamed-list-objects/Makefile

```
@@ -0,0 +1,13 @@
+all: build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)